src/components/Trigger/Trigger.tsx

```diff
@@ -113,28 +113,27 @@ export const Trigger = ({
                         <IconTrashBin size={IconSize.Size12} />
                     </button>
                 )}
-                <div aria-hidden={true}>
-                    <button
-                        {...buttonProps}
-                        type="button"
-                        tabIndex={-1}
-                        disabled={disabled}
-                        className={merge([
-                            'tw-rounded',
-                            disabled
-                                ? 'tw-pointer-events-none tw-text-black-40'
-                                : merge([
-                                      'group-hover:tw-text-black ',
-                                      isOpen ? 'tw-text-black-100' : 'tw-text-black-80',
-                                      FOCUS_VISIBLE_STYLE,
-                                  ]),
-                        ])}
-                    >
-                        <div className={merge(['tw-transition-transform', isOpen && 'tw-rotate-180'])}>
-                            <IconCaretDown size={IconSize.Size16} />
-                        </div>
-                    </button>
-                </div>
+                <button
```

Comment:
Did you check the Axe devTools? I remember I did that change and when reviewing with Lidija, it was apparently necessary to have the aria-hidden on a parent level and not the button.

Reply:
Yup, nothing new appeared